AT/views.py

```diff
@@ -1,106 +1,93 @@
-
-    Request Args:
-        filename: Name of the sample data file
-
-    Returns:
-        data: Json string with the read data
-        columns: array with the columns of the data
-
-    """
-
-    def post(self, request, format=None):
-
-        # Set the path of the file containing data
-        file_path = 'AT/Data/' + request.data['filename']
-
-        data = pd.read_csv(file_path, parse_dates=True, index_col='timestamp')
-
-        return Response(obtain_features(data))
-
-
-class ReadUploadedData(APIView):
-
-    def post(self, request, format=None):
-
-        data = pd.read_csv(request.data['file'], parse_dates=True, index_col='timestamp')
-
-        return Response(obtain_features(data))
-
-
-# Import sample database
-class ImportDatabase(APIView):
-
-    def post(self, request, format=None):
-
-        filePath = 'AT/Databases/' + request.data['filename']
-
-        data = pd.read_csv(filePath, parse_dates=True)
-
-        out = data.to_json(orient='records', date_format='iso')
-
-        return Response(json.loads(out))
-
-
-# Import database from File
-class ImportDatabaseFromFile(APIView):
-
-    def post(self, request, format=None):
-
-        data = pd.read_csv(request.data['file'], parse_dates=True)
-
-        out = data.to_json(orient='records', date_format='iso')
-
-        return Response(json.loads(out))
-
-
-# Removes selected variables from the analysis
-class RemoveVariables(APIView):
-
-    def post(self, request, format=None):
-
-        data = pd.read_json(request.data['data'], orient='records').set_index('timestamp')
-
-        variables = request.data['listSelectedVariables']
-
-        dataOut = data.drop(columns=variables)
-
-        return Response({"data": obtain_features(dataOut),
-                         'writtenResponse': [{'introduction': 'The following variables were removed:',
-                                              'bulletPoints': variables}]})
+import pandas as pd
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from channels.layers import get_channel_layer
+
+from auth_API.helpers import get_or_create_user_information
+from AT.simulator_thread.simulator_routine_by_dummy_eclss import simulate_by_dummy_eclss
+from AT.hub_thread.hub_routine import hub_routine
+from AT.ad_thread.ad_routine import anomaly_detection_routine
+from AT.diagnosis_thread.diagnosis_routine import diagnosis_routine
+
+# QUEUES
+from AT.queue_objects import frontend_to_hub_queue
+from AT.queue_objects import simulator_to_hub_queue
+from AT.queue_objects import hub_to_simulator_queue
+from AT.queue_objects import hub_to_ad_queue
+from AT.queue_objects import ad_to_diagnosis_queue
+from AT.queue_objects import diagnosis_to_hub_queue
+
+
+class SimulateTelemetry(APIView):
+    def post(self, request):
+        # Get the user information and channel layer
+        thread_user_info = get_or_create_user_information(request.session, request.user, 'AT')
```

Comment:
maybe stop past telemetry feeds so everything doesn't crash bad by clicking start twice?

Reply:
Is this fixed?